src/client/OrganizationAccountClient.ts

```diff
@@ -441,8 +441,15 @@ export default class OrganizationAccountClient {
           return response?.data;
         },
       );
-      return repoSecrets || [];
+      return repoEnvSecrets || [];
     } catch (err) {
+      if (err.status === 403) {
+        //this is caused by repos with more restrictive privacy settings
+        this.logger.info(
+          `Repo ${repoName} returned a 403 unauthorized when environmental secrets requested.`,
```

Comment:
Can we change this to have `repoName` as an object property in the first arg? We should avoid string interpolation/concatenation of log messages when possible. Static log messages are easier and faster to search for. 

Searching for an exact match of `"Repo returned a 403 unauthorized when environmental secrets requested."` is faster than leveraging the "contains partial" operator and searching for something like `"returned a 403 unauthorized when environmental secrets requested."`

```suggestion
          { repoName },
          'Repo returned a 403 unauthorized when environmental secrets requested.',
```

Reply:
Great point - since I'm not directly using Cloudwatch, I didn't think of that.